### PR TITLE
Establishes local connection to retrained BERT model

### PR DESCRIPTION
**Pull request by hillarykhan**
Naming: new_bert_model_implementation

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
  
- The Twitter scraper was originally connected to an older version of the BERT model through an API hosted on AWS. We decided to make this more efficient by removing the use of an extra API and allowing the scraper to access the model in a local environment.

  What specific changes were made?
- To establish a local connection we needed to change a few files:
- First, we added frankenbert.py to store the FrankenBert class that allows us to load a serialized BERT model and make predictions on scraped Twitter data.
-  Second, we imported the FrankenBert class to the scraper.py file and replaced the old connection to the API with a local connection
- Third, we updated the clean_data function in helper_funcs.py to better prepare the scraped Twitter data for the loaded BERT model.
- Lastly, we reverted to the sandbox database while making these changes and will need to revert back to the Heroku database on our next commit.

* To note: we were unable to add the file with the serialized BERT model to the repo due to file size constraints. We will need to assess next steps to address this issue.

Submitter Checklist: 
  - [X] Small scope (1-2 components)
  - [X] Not too many concerns in a single commit
  - [X] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [X] Atomic, descriptive commit messages
  - [X] Consistent formatting

Reviewer Checklist:
  - [ ] Small scope (1-2 components)
  - [ ] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [ ] Give feedback no matter what conversations happened elsewhere, document here
  - [ ] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

File: app/pytorch_load.ipynb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ]
+}
```
Comment (ericyeonpark) [inline app/pytorch_load.ipynb:168]: I've made a note to delete this file when we clean our data at the end of the project. I don't think this notebook will help any future teams work on this project.

File: app/scraper.py [hunk 2]
```diff
@@ -15,11 +15,13 @@
...
 from app.TagMaker import TagMaker
 from app.TagList import pb_tags
+from app.frankenbert import FrankenBert
+
 
 # import BD url from .env file
 load_dotenv()
 # make database connection
-db = dataset.connect(os.getenv("DB_URI"))
+db = dataset.connect(os.getenv("DB_URL"))
```
Comment (ericyeonpark) [inline app/scraper.py:24]: To note, we will change back to URI after we automate scraper

**Review by ericyeonpark — APPROVED** [review]
Looks great! Just added comments for my own reflection

**Review by MarcosMorales2011 — APPROVED** [review]
Looks good to me!

**Pull request merged**